src/main/scala/it/pagopa/interop/partyprocess/api/impl/ProcessApiServiceImpl.scala

```diff
@@ -329,7 +329,17 @@ class ProcessApiServiceImpl(
       validManager     <- getValidManager(activeManager, validUsers)
       personsWithRoles <- Future.traverse(validUsers)(addUser)
       relationships    <- Future.traverse(personsWithRoles) { case (person, role, product, productRole) =>
-        createOrGetRelationship(person.id, institution.id, roleToDependency(role), product, productRole)(bearer)
+        val isManager = role == PartyRole.MANAGER
+        createOrGetRelationship(
+          person.id,
+          institution.id,
+          roleToDependency(role),
+          product,
+          productRole,
+          if (isManager) onboardingRequest.pricingPlan else Option.empty,
+          if (isManager) onboardingRequest.institutionUpdate else Option.empty,
+          if (isManager) onboardingRequest.billing else Option.empty
+        )(bearer)
```

Comment:
```suggestion
       role match {
         case PartyRole.MANAGER =>
           createOrGetRelationship(
          person.id,
          institution.id,
          roleToDependency(role),
          product,
          productRole,
          onboardingRequest.pricingPlan,
          onboardingRequest.institutionUpdate,
          onboardingRequest.billing
        )(bearer)
        case _ => 
          createOrGetRelationship(
          person.id,
          institution.id,
          roleToDependency(role),
          product,
          productRole,
          None,
          None,
          None
        )(bearer)
       }
```
sorry for the ugly formatting 😅